app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = []
+    for person in people:
+        name = person["name"]
+        age = person["age"]
+        persons.append(Person(name, age))
+
+    for person in people:
+        name = person["name"]
+        spouse_name = person.get("wife") or person.get("husband")
```

Comment:
Use the `dict.get()` method to access the `name` and `spouse_name` values from the dictionary. This approach is recommended by the checklist to avoid potential `KeyError` exceptions.